docs/fundamentals/dashboard/explore.md

```diff
@@ -180,6 +180,22 @@ For more information and examples of Structured logs, see the [Structured logs p
 >
 > For more information about configuring a resource service, see [Dashboard configuration](configuration.md).
 
+## Interaction prompts
```

Comment:
IMO this section could be displayed near the bottom of the page. Using prompts is fairly self explanitory. The section is more of an FYI in case people are wondering where the UI is coming from.